CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.8.0 (2020-02-18)
+
+- Added `frameOptions` property. [#34](https://github.com/blackbaud/skyux-config/pull/34)
```

Comment:
Do we want to mention `bbCheckout` here?

Reply:
We sure do, good catch.